src/index.html

```diff
@@ -1,13 +1,423 @@
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>MyBike</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#home" class="logo">
+              <img 
+                class="logo__image"
+                src="images/logo.svg" 
+                alt="MyBike">
+            </a>
+            <div class="icons">
+              <a 
+                href="tel:+1 234 555-55-55" 
+                class="icons icons--call"
+              >
+                <div class="icons__tooltip">
+                +1 234 555-55-55
+                </div>
+              </a>
+  
+              <a href="#menu" class="icons icons--menu-opener"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content content">
+          <div class="icons menu-icons menu__top">
+            <a href="#" class="logo-menu">
+              <img 
+                class="logo-menu__image menu-img"
+                src="images/logo__menu.png" 
+                alt="MyBike">
+            </a>
+            <a href="#" class="icons icons--cross"></a>
+          </div>
+            
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+            
+            <li class="menu__item">
+              <a href="#move-free" class="menu__link">About us</a>
+            </li>
+            
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare Bikes</a>
+            </li>
+            
+            <li class="menu__item">
+              <a href="#the-details" class="menu__link">Details</a>
+            </li>
+            
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+          
+          <a 
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--phone"
+            >
+            +1 234 5555-55-55
+          </a>
+          
+          <a 
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--action"
+            >
```

Comment:
Pls, pay attention
Fix it in all cases, in all file
```suggestion
          >
            +1 234 5555-55-55
          </a>
          
          <a 
            href="tel:+1 234 5555-55-55" 
            class="menu__call menu__call--action"
          >
```